contracts/examples/multisig/tests/multisig_rust_test_setup/mod.rs

```diff
@@ -0,0 +1,280 @@
+                        amount,
+                        source,
+                        code_metadata,
+                        arguments,
+                    } => sc.propose_sc_deploy_from_source(
+                        BigUint::from_bytes_be(&amount.to_bytes_be()),
+                        managed_address!(&source),
+                        code_metadata,
+                        boxed_bytes_vec_to_managed(arguments).into(),
+                    ),
+                    ActionRaw::SCUpgradeFromSource {
+                        sc_address,
+                        amount,
+                        source,
+                        code_metadata,
+                        arguments,
+                    } => sc.propose_sc_upgrade_from_source(
+                        managed_address!(&sc_address),
+                        BigUint::from_bytes_be(&amount.to_bytes_be()),
+                        managed_address!(&source),
+                        code_metadata,
+                        boxed_bytes_vec_to_managed(arguments).into(),
+                    ),
+                }
+            },
+        );
+
+        (action_id, tx_result)
+    }
+
+    pub fn call_sign(&mut self, action_id: usize) -> TxResult {
+        self.b_mock.execute_tx(
+            &self.board_member_address,
+            &self.ms_wrapper,
+            &rust_biguint!(0),
+            |sc| {
+                sc.sign(action_id);
+            },
+        )
+    }
+
+    pub fn call_unsign(&mut self, action_id: usize) -> TxResult {
+        self.b_mock.execute_tx(
+            &self.board_member_address,
+            &self.ms_wrapper,
+            &rust_biguint!(0),
+            |sc| {
+                sc.unsign(action_id);
+            },
+        )
+    }
+
+    pub fn call_perform_action(&mut self, action_id: usize) -> TxResult {
+        self.b_mock.execute_tx(
+            &self.board_member_address,
+            &self.ms_wrapper,
+            &rust_biguint!(0),
+            |sc| {
+                let _ = sc.perform_action_endpoint(action_id);
+            },
+        )
+    }
+
+    pub fn call_perform_action_with_result(&mut self, action_id: usize) -> (TxResult, Address) {
+        let mut addr = Address::zero();
+        let tx_result = self.b_mock.execute_tx(
+            &self.board_member_address,
+            &self.ms_wrapper,
+            &rust_biguint!(0),
+            |sc| {
+                let opt_address = sc.perform_action_endpoint(action_id);
+                addr = opt_address.into_option().unwrap().to_address();
+            },
+        );
+
+        (tx_result, addr)
+    }
+
+    pub fn call_discard_action(&mut self, action_id: usize) -> TxResult {
+        self.b_mock.execute_tx(
+            &self.board_member_address,
+            &self.ms_wrapper,
+            &rust_biguint!(0),
+            |sc| {
+                let _ = sc.discard_action(action_id);
+            },
+        )
+    }
+}
+
+fn boxed_bytes_vec_to_managed<M: ManagedTypeApi>(
```

Comment:
I think we have conversion already implemented for that, try .into() or ManagedVec::<M, ManagedBuffer<M>>::from(raw_vec)

Reply:
You can find the From implementation in managed_vec.rs line 78

Reply:
The value required is MultiValueEncoded, which is why I convert separately, and then use `into()` again on the result